mysql/datadog_checks/mysql/mysql.py

```diff
@@ -828,7 +834,33 @@ def _query_size_per_schema(self, db):
 
                 return schema_size
         except (pymysql.err.InternalError, pymysql.err.OperationalError) as e:
-            self.warning("Avg exec time performance metrics unavailable at this time: %s", e)
+            self.warning("Size of schemas metrics unavailable at this time: %s", e)
+
+        return {}
+
+    def _query_size_per_table(self, db):
+        try:
+            with closing(db.cursor()) as cursor:
+                cursor.execute(SQL_QUERY_TABLE_SIZE)
+
+                if cursor.rowcount < 1:
+                    self.warning("Failed to fetch records from the information schema 'tables' table.")
+                    return None
+
+                table_index_size = {}
+                table_data_size = {}
+                for row in cursor.fetchall():
+                    table_name = str(row[0])
+                    index_size = long(row[1])
+                    data_size = long(row[2])
+
+                    # set the tag as the dictionary key
+                    table_index_size["schema:{0}".format(table_name)] = index_size
+                    table_data_size["schema:{0}".format(table_name)] = data_size
```

Comment:
I'm not sure how the schema & table name should be stored.
Here, you defining one tag called `schema` and the value is concatenating the 2 (`my_schema.my_table`), which is not very straightforward. Maybe, it should be 2 different tags: `schema` & `table` (to be able to filter more precisely when reading metrics later), but I don't know if it's possible in the datadog "framework".

Reply:
@hithwen might be able to help here

Reply:
@jmeunier28 : what will be the generated metrics (which labels will they have) ?

Reply:
Here with the current code the generated label should be `schema`: <my_schema.my_table>

Reply:
@jmeunier28 Can you confirm the workflow and the **expected label with the current code**?

Also it would be very nice to generate 2 separate labels `schema` and `table`, we would appreciate if you can provide a suggestion on how to achieved this as it is hard to test on our side thanks

Reply:
> Also it would be very nice to generate 2 separate labels schema and table, we would appreciate if you can provide a suggestion on how to achieved this as it is hard to test on our side thanks

I would suggest you write some tests for this change. You can set up [ddev](https://datadoghq.dev/integrations-core/setup/), which would allow you to set up a local mysql environment which would give you more insights into the metrics you are producing 

e.g I started to do this here https://github.com/DataDog/integrations-core/commit/de732b6d09b137145db974d41959cecf7e1f2187

I set my environment up like so:
```
virtualenv env --python=python3.8
python -m pip install --upgrade "datadog-checks-dev[cli]"
ddev env ls mysql (gives you a list of envs)
ddev env start mysql py38-5.7

# once the env is running you can run a single test like this:
ddev test mysql:py38-5.7,style --skip-env -k test_complex_config

or the entire suite like this:
ddev test mysql:py38-5.7,style
```

I don't have time left today to continue on with testing, but I think it would helpful for you to get more visibility into what is happening (also when I add your metric one of the test cases fails so i'll look into that tomorrow)

As for making `schema` and `table` separate tags, that should be fine to do. If you update your query to grab both instead of doing `CONCAT(table_schema,'.',table_name) as table_name,` you can set them as separate tags like so:

```
table_index_size["table:{}".format(table_name)]
table_index_size["schema:{}".format(schema_name)]
table_data_size["schema:{}".format(schema_name)]
table_data_size["table:{}".format(table_name)]
```

The code is a little hard to follow, but essentially what is happening is when we call [__submit_metrics](https://github.com/DataDog/integrations-core/blob/703fd453123c5075ae27aecdf37723fa0e0a4ee4/mysql/datadog_checks/mysql/mysql.py#L501) at the end it uses [this method](https://github.com/DataDog/integrations-core/blob/e773f597247510fa8304de2c78abdfd49955f22a/mysql/datadog_checks/mysql/collection_utils.py#L11) to convert the keys in the dictionary to tags

Reply:
OK, this commit fixes the test https://github.com/DataDog/integrations-core/commit/6bf3e0c10f8c0d9ca05874c52151d6450fd28c45, and it also makes a change to create schema and table tags

Reply:
The PR currently implements @jmeunier28's suggestion:
```
                    table_index_size["schema:{0}".format(table_schema)] = index_size
                    table_index_size["table:{0}".format(table_name)] = index_size
                    table_data_size["schema:{0}".format(table_schema)] = data_size
                    table_data_size["table:{0}".format(table_name)] = data_size
```
I don't really understand how it will work, but if it's the recommended way, it's ok for me 👍